doc/HOW-TO/docker-build.rst

```diff
@@ -0,0 +1,29 @@
+Build a Qserv container image
+############################################################
```

Comment:
Headings need both upper and under lines. Lines need to be the same length as the content.

For more information, see the [ReStructuredText Style Guide on headings](https://developer.lsst.io/restructuredtext/style.html#sections)

Reply:
:+1: